accepted/2024/relax-dotnet-foundation-requirements.md

```diff
@@ -0,0 +1,77 @@
+# Relax signing certificate requirements for the dotnetfoundation user
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Author Name @glennawatson
+- Issue https://github.com/NuGet/NuGetGallery/issues/10187
+
+## Summary
+
+At the moment the .NET foundation user (dotnetfoundation) on nuget.org has control over packages signing permissions. Users can't be the author signing owner of the package and therefore maintainers can't bring their own certificates for author singing.
+
+## Motivation 
+
+The foundation is having complaints from maintainers that they can't control the signing permissions. Also we want the ability for projects to use other signing tech if they want.
+
+We do want to have the project to have some sort of signing since we offer projects free signing certificates to mitigate any costs to the project.
+
+We have had discussions and approval by the .NET Foundation board, and also discussed with our Project Committe and they have agreed to these requirements.
```

Comment:
I hope this isn't a large ask, but is there a ledger of the votes anywhere so we have a reference of the agreement? Or just other people vouching here. There's upvotes, are those people from these groups?

Thanks!

Reply:
Here's meeting notes i found online:

https://github.com/dotnet-foundation/projects/issues/398 would be nice to have that included here.

Reply:
Worth noting that the upvotes are almost exclusively from .NET Foundation maintainers/board members or project committee members.

I've reached out the project committee and the board to make attestations